README.rst

```diff
@@ -77,6 +77,19 @@ Project goals
   ``xarray.DataArray`` for grids.
 
 
+Quickstart
+----------
+
+Installation
+++++++++++++
```

Comment:
Good catch. I've put in two separate section headers because I'm thinking of adding a small code snippet below the installation as suggested in https://github.com/pyOpenSci/software-review/issues/43#issuecomment-1003750492

> * [x]  **Vignette(s)** demonstrating major functionality that runs successfully locally
> 
> 
> Not provided in a textual form but linked as a tutorial video, so for me, it's ok. However, I think that the short copy & paste example below the clip could be applicable for more advanced users who would like to test the package quickly. For example, it could look like:
> 
>     * Youtube lecture,
> 
>     * Short installation,
> 
>     * Copy/Paste example.

Reply:
Sounds good, nice planning 😄